db/schema.rb

```diff
@@ -313,12 +318,12 @@
     t.string "last_sign_in_ip"
     t.integer "role"
     t.string "old_user_id"
+    t.string "phone"
```

Comment:
Don't think the phone field order swapping should be here

Reply:
That was done automatically after running migration scripts, I don't think it would have any impact but would you suggest removing the change?

Reply:
Yep. As a general rule of thumb you only want to be committing schema changes if your PR has a migration that is expected to introduce them or if you've run migrations on a fresh database and got them (i.e. main schema is already incorrect/someone has forgotten to commit).

Change like this is likely just introduced on your local from running migrations in a slightly different order (not rolling back when working on a PR etc) so you just get back and forth on each PR.